recipes/oatpp/all/conanfile.py

```diff
@@ -59,6 +64,8 @@ def generate(self):
         if is_msvc(self) and Version(self.version) >= "1.3.0":
             tc.variables["OATPP_MSVC_LINK_STATIC_RUNTIME"] = is_msvc_static_runtime(self)
         tc.generate()
+        venv = VirtualBuildEnv(self)
```

Comment:
We also probably want to set `OATPP_LINK_TEST_LIBRARY` to false in the `CMakeToolchain` here

Reply:
@RubenRBS 
Thanks a lot!
I add `with_test_library` option and use it's value for OATPP_LINK_TEST_LIBRARY.